src/main/java/frc/robot/Constants.java

```diff
@@ -259,6 +184,8 @@ public static class GamePieceManipulatorConstants {
     public static final AngularVelocity EJECT_ALGAE_VELOCITY = RadiansPerSecond.of(-5);
     public static final AngularVelocity SCORE_ALGAE_VELOCITY = RadiansPerSecond.of(-5);
 
+    public static final AngularVelocity ROLLER_SPEED_TOLERANCE = RotationsPerSecond.of(3);
```

Comment:
What you did is good, but I see there's some more cleanup that can happen here.
There are duplicate constants for intake, eject, and score. We should keep the `_VELOCITY` fields and we can get rid of the `_SPEED` fields.
Also, the `ROLLER_SPEED_TOLERANCE` should be renamed because it's the speed of the wheels, not the rollers.